infra/ansible/roles/django/tasks/main.yml

```diff
@@ -238,3 +238,22 @@
     state: "{{ 'started' if tournesol_twitterbot_schedule is defined else 'stopped' }}"
     enabled: "{{ 'yes' if tournesol_twitterbot_schedule is defined else 'no' }}"
     daemon_reload: yes
+
+# scheduled task: Sync Youtube playlists
+
+- name: Copy tournesol-yt-playlists service
+  template:
+    dest: /etc/systemd/system/tournesol-yt-playlists.service
+    src: tournesol-yt-playlists.service.j2
```

Comment:
I haven't found this file in this PR, maybe it's not commited yet?

Reply:
These 2 files had not committed. Sorry for that :clown_face:   
Fixed in cbd9a46